### DISCUSSION TITLE
Possible to get an iOS version?

### DISCUSSION THREAD

**Discussion by foundationpineapple** [feelfreelinux/octo4a#91] [Ideas]
I know nothing so I am not sure it's even feasible but I wanted to know if it was possible to get a version that would work on iPhones. Jailbroken or not. I just upgraded from my iphone 11 and would love to have that camera used for octolapse.

**Comment by Ferdous123** [comment]
Exactly. If devs could do this it would be so cool. Since every Phone till ios 12 can be now jailbroken it’s not a big deal. 
Knowing nothing about these, I don’t if it’s possible or not. Let’s just wait till the devs comment on it...

**Comment by sbaeder** [comment]
A lot of this is because Android is LINUX under the covers, with a lot of layers on top of it.  So, just like any "linux" os, it leverages that basis to install and run Python and all the other dependencies that are needed to run Octoprint, which is written in Python.

This project (to the best of my limited knowledge) builds on and automates a lot of the process that existed before to run an alternate "Linux environment" and installs octoprint on top of that.  And since it is all "arm" based, this isn't that difficult.

My guess is that since iOS (while based on arm) is going to be different, and since it doesn't have "linux" at the core, it would be a lot more effort.  On the other hand, if you can get a python environment setup, it might be possible to make OctoPrint work...
